tapscript/interface.go

```diff
@@ -17,6 +17,15 @@ type TxValidator interface {
 		prevAssets commitment.InputSet) error
 }
 
+// VPsbtValidator is the interface used to validate a vPSBT. It is less strict
+// compared to the TxValidator, as it focused on witness validation. This method
+// may be used in partially constructed transfers.
+type VPsbtValidator interface {
+	ValidateVPsbt(newAsset *asset.Asset,
```

Comment:
nit: interface methods also need Godoc comments. Even if they are kind of identical to the interface comment for 1-method interfaces.